documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -168,6 +168,14 @@ If you are using a different DNS domain name suffix in your cluster, change the
 The timeout for building new Kafka Connect images with additional connectots, in milliseconds.
 This value should be increased when using Strimzi to build container images containing many connectors or using a slow container registry.
 
+`STRIMZI_NETWORK_POLICY_GENERATION` :: Optional, default `true`.
+The boolean value, controlling whether Strimzi should generate Network Policy files using the operator.
```

Comment:
Maybe `Network Policy resources` is better than `Network Policy files`?

Reply:
Switched to better naming.